src/arviz_plots/backend/__init__.py

```diff
@@ -99,6 +99,33 @@ def create_plotting_grid(
 
 
 # "geoms"
+def hist(y, l_e, r_e, target, *, bottom=None, color=None, facecolor=None, edgecolor=None):
```

Comment:
I went over all potential keywords in https://arviz-plots.readthedocs.io/en/latest/api/backend/index.html#backend-interface-arguments and `alpha` should be added here too.